Namco/map.py

```diff
@@ -1,24 +1,19 @@
 import numpy as np
+from bosch_ASDIIE_Namco.Namco.constants import Constants
```

Comment:
Ha az osztálynak átadjuk az enumot, akkor lehet ez nem is kell majd ide, hanem csak a game_runner-be. De ezt meg kell nézni, meg a PyCharm úgyis sír majd érte